.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "rust-analyzer.cargo.target": "i686-pc-windows-gnu"
+    "rust-analyzer.cargo.target": "i686-pc-windows-msvc"
```

Comment:
unlikely spacemaniac will accept this, per the issue

Reply:
he didnt comment on the issue though

Reply:
This applies to literally everyone else though, and we shouldn't impact the majority for the sake of one person.

Reply:
this made it so much more painful to start guys